submissions/migrations/0003_submission_status.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('submissions', '0002_auto_20151119_0913'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='submission',
+            name='status',
+            field=models.CharField(default=b'A', max_length=1, choices=[(b'D', b'Deleted'), (b'A', b'Active')]),
+        ),
+    ]
```

Comment:
@maxrothman (and FYI @scottrish), here's the timing data I got from my sandbox. 90 seconds to apply the migration to a table with 700,000 rows.

```
mysql> show create procedure stuff_submissions_tables;
+--------------------------+--------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------+----------------------+--------------------+
| Procedure                | sql_mode                                   | Create Procedure                                                                                                                                                                                                                                                                                                                                                                                       | character_set_client | collation_connection | Database Collation |
+--------------------------+--------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------+----------------------+--------------------+
| stuff_submissions_tables | STRICT_TRANS_TABLES,NO_ENGINE_SUBSTITUTION | CREATE DEFINER=`root`@`localhost` PROCEDURE `stuff_submissions_tables`(in numRows int)
begin
declare i int;
set i = 1;
start transaction;
while i < numRows do
insert into submissions_studentitem values(i, uuid(), uuid(), uuid(), "openassessment");
insert into submissions_submission values (i, uuid(), 1, now(), now(), concat(uuid(), uuid(), uuid()), i);
set i = i+1;
end while;
commit;
end | utf8                 | utf8_general_ci      | utf8_general_ci    |
+--------------------------+--------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------+----------------------+--------------------+
1 row in set (0.00 sec)

mysql> call stuff_submissions_tables(700000);
Query OK, 0 rows affected (1 min 41.62 sec)

mysql> select count(*) from submissions_submission;
+----------+
| count(*) |
+----------+
|   699999 |
+----------+
1 row in set (0.14 sec)

mysql> exit
Bye
[efischer19-sandbox] efischer19@efischer19 i-6fd3e6ea:~$ time sudo -u www-data /edx/bin/python.edxapp /edx/bin/manage.edxapp lms --settings aws migrate submissions 0003

Operations to perform:
  Target specific migration: 0003_submission_status, from submissions
Running migrations:
  Rendering model states... DONE
  Applying submissions.0003_submission_status... OK

real	1m29.847s
user	0m3.460s
sys	0m0.412s
```